build-system/pr-check/build-targets.js

```diff
@@ -252,6 +281,12 @@ function determineBuildTargets(fileName = 'build-targets.js') {
         .join(', ')
     )
   );
+  if (isTravisBuild() && buildTargets.has('PACKAGE_UPGRADE')) {
+    console.log(
+      `${fileLogPrefix} Since this PR contains package upgrade(s), running all checks...`
+    );
+    ALL_TARGETS.forEach(target => buildTargets.add(target));
```

Comment:
Can we move this to the beginning of `determineBuildTargets` for an earlier return?

Reply:
Can't do so because `buildTargets` needs to be populated first in order to determine if it contains `PACKAGE_UPGRADE`.

There's also a logging benefit in doing this at the end. Printing the originally detected targets first will help debug any erroneous detection.